simplematrixbotlib/match.py

```diff
@@ -83,7 +85,27 @@ def __init__(self, room, event, bot, prefix="") -> None:
         super().__init__(room, event, bot)
         self._prefix = prefix
 
-    def command(self, command=None):
+        """Forms of identification"""
+        self._own_user_id = room.own_user_id
+        self._own_nio_user = self.room.users[self._own_user_id]
+        self._own_disambiguated_name = self._own_nio_user.disambiguated_name
+        self._own_display_name = self._own_nio_user.display_name
+        self._own_display_name_colon = f"{self._own_display_name}:"
+        self._own_pill = f"<a href=\"https://matrix.to/#/{self._own_user_id}\">"
+
+        self.mention() # Set self._mention_id_length
+        self._body_without_prefix = self.event.body[len(self._prefix):]
+        self._body_without_mention = self.event.body[self._mention_id_length:]
```

Comment:
why initialize these here while it's unknown whether prefix or mention is used?

Reply:
mention() sets `._mention_id_length`, which is neccesary for calculating `._body_without_mention`

Reply:
I think it may be a good idea to look at bot libraries for other networks to see how they handle matching like this.